src/init.cpp

```diff
@@ -893,6 +895,11 @@ bool AppInitParameterInteraction(const ArgsManager& args)
         InitWarning(_("Options '-datacarrier' or '-datacarriersize' are set but are marked as deprecated. They will be removed in a future version."));
     }
 
+    // We no longer limit the orphanage based on number of transactions but keep the option to warn users who still have it in their config.
+    if (args.IsArgSet("-maxorphantx")) {
+        InitWarning(_("Option '-maxorphantx' is set but no longer has any effect (see release notes). Please remove it from your configuration."));
+    }
```

Comment:
Might be nice to set `-maxorphantx` in one functional test to ensure it doesn't error.

Reply:
added